docs/api.yml

```diff
@@ -525,11 +548,20 @@ components:
     NoteLink:
       type: object
       properties:
+        type:
+          type: string
+          example: splash
```

Comment:
I think this should probably say <type of content available at the URL, e.g. "fulltext" or "package" or "splash">